components/input-validation-mgt/org.wso2.carbon.identity.input.validation.mgt/src/main/java/org/wso2/carbon/identity/input/validation/mgt/model/validators/AlphanumericValidator.java

```diff
@@ -99,6 +105,14 @@ public List<Property> getConfigurationProperties() {
         enableValidator.setDisplayOrder(++parameterCount);
         configProperties.add(enableValidator);
 
+        Property allowSpecialChars = new Property();
+        allowSpecialChars.setName(ENABLE_SPECIAL_CHARACTERS);
+        allowSpecialChars.setDisplayName("Alphanumeric field value with special characters");
+        allowSpecialChars.setDescription("Validate whether the field value is alphanumeric.");
```

Comment:
The description should also mention about special characters right?

Reply:
Addressed with https://github.com/wso2/carbon-identity-framework/pull/5203/commits/b94e4a7e28c895dbb6faf9fd421f9941a4722dad